docs/content/en/blog/0.2.0_release.md

```diff
@@ -0,0 +1,7 @@
+---
+title: "Kurator v0.2.0 is live!"
+date: 2023-02-09
+linkTitle: "Kurator v0.2.0 is live!"
+---
+
+Kurator v0.2.0 adds observability and cluster lifecycle management, featuring multi-cluster monitoring with Thanos, real-time K8s app monitoring with Pixie, and lifecycle management for local data center and AWS-built clusters, enhancing distributed cloud-native infrastructure management.
```

Comment:
Add a link to the detailed note

Reply:
Apologies for not being clear enough. 

The purpose of this PR is to create a blog page, and the current articles are just placeholders to demonstrate the blog layout. 

I will submit complete articles later to replace these placeholder posts. 

This is the full plan https://github.com/kurator-dev/kurator/issues/452